urwid/event_loop/asyncio_loop.py

```diff
@@ -57,14 +59,20 @@ class AsyncioEventLoop(EventLoop):
             asyncio.get_event_loop().call_soon(main_loop.draw_screen)
     """
 
-    _we_started_event_loop = False
-
     def __init__(self, *, loop: asyncio.AbstractEventLoop | None = None, **kwargs) -> None:
         super().__init__()
         self.logger = logging.getLogger(__name__).getChild(self.__class__.__name__)
         if loop:
             self._loop: asyncio.AbstractEventLoop = loop
+            self.__event_loop_policy_altered: bool = False
```

Comment:
double-underscore will munge the name and make it more difficult to update/access from a subclass. Can we use a single underscore instead?

Reply:
Better to not allow such changes from outside

Reply:
I disagree because there are no real private/protected variables in python so it's better to indicate your intent with a single underscore than to try and fake it with a munged name.

If someone wants to override our values from their code this won't prevent them and there's no reason for us to assume we know better what they need in their particular environment/application.

Reply:
not fully agree, but removed mangle